jekyll-exporter.php

```diff
@@ -127,15 +127,15 @@ function get_posts() {
 		}
 
 		$posts = array();
-		$post_types = apply_filters( 'jekyll_export_post_types', array( 'post', 'page' ) );
+		$post_types = apply_filters( 'jekyll_export_post_types', array( 'post', 'page', 'revision' ) );
```

Comment:
Will this return all revisions (including those for published posts) or only revisions for unpublished posts?

Reply:
all revisions, but I don't think it's a bad thing. To my mind, if WP deemed something important enough to keep, so should the exporter (why risk missing something.. one could realize it too late after one has deleted one's original WP DB). It's just going to sit in the `_drafts` folder so no harm done anyway, worst case simply delete what you don't want from there after the export...